.travis.yml

```diff
@@ -42,11 +41,16 @@ before_install:
     ${run_opts} ${distribution}-${version}:ansible "${init}"
 
 install:
-  - 'pip install -r requirements-devel.txt'
+  - 'sudo pip install -r requirements-devel.txt'
```

Comment:
Since it's inside the virtualenv, I think it's better to avoid using `sudo` (which will also suppress [this](https://travis-ci.org/openstack-ansible-galaxy/openstack-keystone/jobs/147701489#L712-L713) warning).